### PR TITLE
1.4.7 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux2-demos?_sp.spa=pr-1.4.7-release](https://host.nxt.blackbaud.com/skyux2-demos?_sp.spa=pr-1.4.7-release)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 1.4.7 (2020-05-07)
+
+- Fixed the colorpicker demo to no longer have redundant examples. [#44](https://github.com/blackbaud/skyux2-demos/pull/44)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: How about "to remove" instead of "to no longer have"?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Works for me. Fixed!

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux2-demos?_sp.spa=pr-1.4.7-release](https://host.nxt.blackbaud.com/skyux2-demos?_sp.spa=pr-1.4.7-release)

**Pull request merged**